Fix CI issue `TAR_BAD_ARCHIVE`

[The issue](https://github.com/Sage-Bionetworks/challenge-registry/runs/5520288017?check_suite_focus=true) started today. It only affects when merging to the `main` branch..

```console
Run yarn nx affected --target=lint --parallel --max-parallel=3
yarn run v1.22.17
$ /home/runner/work/challenge-registry/challenge-registry/node_modules/.bin/nx affected --target=lint --parallel --max-parallel=3

 >  NX   No explicit --base argument provided, but found environment variable NX_BASE so using its value as the affected base: 367f0b1677822ea9b3d6a3bc103b672b2e7e53e7


 >  NX   No explicit --head argument provided, but found environment variable NX_HEAD so using its value as the affected head: 9ec712f6f7259edeb844acdd5515b3a12af37f2c


 >  NX   Running target lint for 12 project(s):

    - web-app
    - web-app-e2e
    - web-config
    - web-pages
    - web-feature-home
    - web-ui
    - api
    - api-angular
    - api-spec
    - db-cli
    - shared-web-components
    - web-data-access

 
events.js:377
      throw er; // Unhandled 'error' event
      ^

Error: TAR_BAD_ARCHIVE: Unrecognized archive format
    at Unpack.warn (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/warn-mixin.js:19:40)
    at Unpack.warn (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/unpack.js:220:18)
    at Unpack.<anonymous> (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:81:14)
    at Unpack.emit (events.js:412:35)
    at Unpack.[emit] (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:285:12)
    at Unpack.[maybeEnd] (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:401:17)
    at Unpack.[consumeChunk] (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:432:21)
    at Unpack.write (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:364:25)
    at Unpack.end (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:477:14)
    at IncomingMessage.onend (internal/streams/readable.js:670:10)
Emitted 'error' event on Unpack instance at:
    at Unpack.onerror (internal/streams/readable.js:750:14)
    at Unpack.emit (events.js:400:28)
    at Unpack.warn (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/warn-mixin.js:19:12)
    at Unpack.warn (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/unpack.js:220:18)
    [... lines matching original stack trace ...]
    at Unpack.write (/home/runner/work/challenge-registry/challenge-registry/node_modules/tar/lib/parse.js:364:25) {
  recoverable: false,
  cwd: '/home/runner/work/challenge-registry/challenge-registry/node_modules/.cache/nx',
  code: 'TAR_BAD_ARCHIVE',
  tarCode: 'TAR_BAD_ARCHIVE'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,7 +10,8 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
-  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+  NX_CLOUD_AUTH_TOKEN: 'plop'
+  # NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -58,8 +58,6 @@ jobs:
   pr:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'pull_request' }}
-    env:
-      NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
     steps:
       - uses: actions/checkout@v2
         with:
@@ -96,5 +94,11 @@ jobs:
         run: |
           yarn nx prepare api
       - run: yarn nx affected --target=lint --parallel --max-parallel=3
+        env:
+          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
       - run: yarn nx affected --target=build --parallel --max-parallel=3
+        env:
+          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
       - run: yarn nx affected --target=test --parallel --max-parallel=2
+        env:
+          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,8 +17,6 @@ jobs:
   main:
     runs-on: ubuntu-latest
     if: ${{ github.event_name != 'pull_request' }}
-    env:
-
     steps:
       - uses: actions/checkout@v2
         name: Checkout [main]

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,8 +10,7 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
-  NX_CLOUD_AUTH_TOKEN: ''
-  # NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,7 +10,8 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
-  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+  NX_CLOUD_AUTH_TOKEN: ''
+  # NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,6 +10,7 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
+  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:
@@ -17,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event_name != 'pull_request' }}
     env:
-      NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+
     steps:
       - uses: actions/checkout@v2
         name: Checkout [main]
@@ -94,11 +95,5 @@ jobs:
         run: |
           yarn nx prepare api
       - run: yarn nx affected --target=lint --parallel --max-parallel=3
-        env:
-          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
       - run: yarn nx affected --target=build --parallel --max-parallel=3
-        env:
-          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
       - run: yarn nx affected --target=test --parallel --max-parallel=2
-        env:
-          NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,13 +10,14 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
-  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:
   main:
     runs-on: ubuntu-latest
     if: ${{ github.event_name != 'pull_request' }}
+    env:
+      NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
     steps:
       - uses: actions/checkout@v2
         name: Checkout [main]
@@ -57,6 +58,8 @@ jobs:
   pr:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'pull_request' }}
+    env:
+      NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
     steps:
       - uses: actions/checkout@v2
         with:

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -10,8 +10,7 @@ env:
   PRIMARY_PYTHON_VERSION: 3.9
   NX_BRANCH: ${{ github.event.number }}
   NX_RUN_GROUP: ${{ github.run_id }}
-  NX_CLOUD_AUTH_TOKEN: 'plop'
-  # NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+  NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
   NX_CLOUD_ENV_NAME: 'linux'
 
 jobs:
@@ -28,7 +27,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.PRIMARY_NODE_VERSION }}
-      - name: Setup node_modules cache
+      - name: Setup Node.js cache
+        id: cache-node
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
@@ -44,7 +44,9 @@ jobs:
         with:
           path: ~/.local/share/virtualenvs
           key: ${{ runner.os }}-pipenv-${{ hashFiles('**/Pipfile.lock') }}
-      - run: yarn install --frozen-lockfile
+      - name: Install Node.js dependencies
+        if: steps.cache-node.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
       - name: Install Python dependencies
         if: steps.cache-pipenv.outputs.cache-hit != 'true'
         run: |
@@ -66,7 +68,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.PRIMARY_NODE_VERSION }}
-      - name: Setup node_modules cache
+      - name: Setup Node.js cache
+        id: cache-node
         uses: actions/cache@v2
         with:
           path: '**/node_modules'
@@ -82,7 +85,9 @@ jobs:
         with:
           path: ~/.local/share/virtualenvs
           key: ${{ runner.os }}-pipenv-${{ hashFiles('**/Pipfile.lock') }}
-      - run: yarn install --frozen-lockfile
+      - name: Install Node.js dependencies
+        if: steps.cache-node.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
       - name: Install Python dependencies
         if: steps.cache-pipenv.outputs.cache-hit != 'true'
         run: |

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159264;
+$pi: 3.14159265;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159265;
+$pi: 3.14159264;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159266111222;
+$pi: 3.141592661112222;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.1415926611122222;
+$pi: 3.14159264;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.141592661;
+$pi: 3.1415926611;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159266111;
+$pi: 3.141592661112;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.141592661112;
+$pi: 3.14159266111222;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159265;
+$pi: 3.14159266;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.141592661112222;
+$pi: 3.1415926611122222;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.1415926611;
+$pi: 3.14159266111;
 $padding-test: 20px;

--- a/libs/shared/style/src/_variables.scss
+++ b/libs/shared/style/src/_variables.scss
@@ -1,5 +1,5 @@
 // See https://github.com/angular/components/blob/master/src/material/core/style/_variables.scss
 
 // Global constants
-$pi: 3.14159266;
+$pi: 3.141592661;
 $padding-test: 20px;

